README.md

```diff
@@ -5,87 +5,57 @@
 
 Fast and embedded solvers for nonlinear optimal control.
 
-### Installation
-If you are on Ubuntu (tested with 16.04), you can run `./install.sh`. You can also
-follow the [manual installation instructions](#manual-installation) below.
-
-* soon: binaries for all operating systems available for download (see Releases)
-
-### Getting started
-
-#### MATLAB
-First, add CasADi and acados to your MATLAB path. From a MATLAB command window
-```
-addpath <path_to_acados_root_folder>/external/casadi-matlabR2014b-v3.4.0
-addpath <path_to_acados_installation_folder>
-% To permanently add these paths:
-savepath
-```
-Run an acados example, from `<path_to_acados_root_folder>/examples/matlab/`:
-```
-ocp_qp_example.m
-```
-
-#### Python
-
-acados only supports `Python3`. Add CasADi and acados to the `PYTHONPATH` environment variable (add those lines to your `.bashrc` or `.zshrc` to set the paths permanently):
-```
-export PYTHONPATH=<path_to_acados_root_folder>/external/casadi-py35-v3.4.0-64bit:$PYTHONPATH
-export PYTHONPATH=<path_to_acados_installation_folder>:$PYTHONPATH
-```
-To run a Python example from the acados root folder:
-```
-python examples/python/ocp_nlp.py
-```
-
-### Manual Installation
+### Optional requirements
+Some functionalities in acados require CasADi (version 3.4.0) to be installed on your system.
```

Comment:
it should work for any version later than 3.4.0 (to be checked)

Reply:
I just checked from octave, and it works with 3.4.5 (but the scripts generating the casadi functions need to be updated, because now they exit if the version is not 3.4.0)

Reply:
Yes but the python interface from @zanellia may be stricter about the required version, any is very easy to get most tested/supported version using the proper link.

Reply:
let's just merge this? We can update it later on ;)